client/lib/user-settings/index.js

```diff
@@ -1,16 +1,9 @@
 /**
  * External dependencies
  */
+import { assign, isEmpty, keys, merge, has, get, set, unset } from 'lodash';
```

Comment:
Before I comment on the other comments below: 
Thanks for your review, highly appreciated! 💯